README.md

```diff
@@ -7,22 +7,26 @@ You will need to have the following:
 - Installed [MongoDB](https://docs.mongodb.com/manual/installation/)
 - Installed/configured [service manager v2](https://github.com/hmrc/sm2).
 
+This can be found in the [developer handbook](https://docs.tax.service.gov.uk/mdtp-handbook/documentation/developer-set-up/)
+
 The service manager profile for this service is:
 
     sm2 --start INCOME_TAX_SUBMISSION_FRONTEND
 Run the following command to start the remaining services locally:
 
-    sudo mongod (If not already running)
-    sm2 --start INCOME_TAX_SUBMISSION_ALL -r
+    sm2 --start INCOME_TAX_SUBMISSION_ALL
 
 This service runs on port: `localhost:9302`
 
+To test the branch you're working on locally. You will need to run `sm2 --stop INCOME_TAX_SUBMISSION_FRONTEND` followed by
+`./run.sh`
+
 ### Running Tests
 - Run Unit Tests:  `sbt test`
 - Run Integration Tests: `sbt it/test`
 - Run Unit and Integration Tests: `sbt test it/test`
-- Run Unit and Integration Tests with coverage report: `sbt runAllChecks`<br/>
-  which runs `clean compile scalastyle coverage test it/test coverageReport`
+- Run Unit and Integration Tests with coverage report and dependency updates: `./check.sh`<br/>
+  which runs `sbt clean scalastyle coverage test it/test coverageReport dependencyUpdates`
 
 ### Feature Switches
 | Feature                | Environments Enabled In            |
```

Comment:
I would remove the `Environment Enabled in` because this would be a nightmare to maintain in here.

Instead, I would just have a description of what the feature switches do (although that might be easier said than done).

Reply:
Noticed we're up to 31 feature switches now in this service, think this is going to take me a good while